platform/ios/src/MGLMapView.mm

```diff
@@ -1572,14 +1572,14 @@ - (void)handlePinchGesture:(UIPinchGestureRecognizer *)pinch
         {
             self.mbglMap.setZoom(newZoom, mbgl::ScreenCoordinate { centerPoint.x, centerPoint.y });
             // The gesture recognizer only reports the gesture’s current center
-            // point, so use the previous center point to anchor the transition.
+            // point, so use the previous center point to centerPoint the transition.
             // If the number of touches has changed, the remembered center point is
             // meaningless.
             if (self.userTrackingMode == MGLUserTrackingModeNone && pinch.numberOfTouches == _previousPinchNumberOfTouches)
             {
                 CLLocationCoordinate2D centerCoordinate = _previousPinchCenterCoordinate;
                 self.mbglMap.setLatLng(MGLLatLngFromLocationCoordinate2D(centerCoordinate),
-                                    mbgl::ScreenCoordinate { centerPoint.x, centerPoint.y });
+                                    mbgl::EdgeInsets { centerPoint.y, centerPoint.x, self.size.height - centerPoint.y, self.size.width - centerPoint.x });
```

Comment:
What's this line doing?

Reply:
It’s still adjusting the frame of reference for the center coordinate, but now it does so by padding all four sides so that the padding covers the entire view, instead of setting an anchor point explicitly.

Reply:
Yes - as @1ec5 mentioned, this was previously done inside an overload for `setLatLng` in `Transform` that also received an anchor point as a parameter - this overload has been removed in this PR to avoid confusion.